lib/runPrettier.js

```diff
@@ -6,7 +6,7 @@ const { runBin } = require('./runBin');
 const { getPathFromCwd } = require('./cwd');
 const suggestScript = require('./suggestScript');
 
-const exists = promisify(fs.access);
+const exists = promisify(fs.stat);
```

Comment:
The promisify-ing of `access` was not correct, meaning a custom prettierignore path would not resolve, more of an impact on the test suite than consumers.

Reply:
We should just move to `fs.promises` across the board.

Reply:
Agree, but definitely a separate discreet piece of work as it involves moving to node 12

Reply:
Not necessarily. You can just use it in 10 and turn on the `--experimental-modules` flag

Reply:
But fair enough not part of this pr

Reply:
I know we **can** but i dont think sku **should** be setting that flag personally.